movies/2013/sharknado.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Sharknado",
+    "year": 2013,
+    "runtime":86,
+    "categories":[
+        "action",
+        "adventure",
+        "comedy"
+    ],
+    "release-date":"2013-07-11",
+    "director":"Anthony C. Ferrante",
+    "writer":[
+        "Thunder Levin"
+    ],
+    "actors":[
+        "Ian Ziering",
+        "Tara Reid",
+        "John Heard",
```

Comment:
This comma is making this file not be a valid json file.

Reply:
@avdLuna use http://jsonlint.com to validate json